cmd/utils/flags.go

```diff
@@ -790,59 +790,31 @@ func MustMakeChainConfigFromDb(ctx *cli.Context, db ethdb.Database) *core.ChainC
 	c := new(core.ChainConfig)
 
 	genesis := core.GetBlock(db, core.GetCanonicalHash(db, 0))
-	if genesis != nil {
-		storedConfig, err := core.GetChainConfig(db, genesis.Hash())
```

Comment:
what was the reason for writing ChainConfig to db previously? I don't see a problem with these changes, I'm just wondering why both were used before.